src/InvoicePrinter.php

```diff
@@ -27,6 +27,7 @@ class InvoicePrinter extends FPDF
         'r' => 15
     ]; /* l: Left Side , t: Top Side , r: Right Side */
 
+    public $charset;
```

Comment:
I would make this proteced or private since we have a setter, and no one uses it outside of the class.